github/GithubObject.py

```diff
@@ -182,6 +183,8 @@ def parseDatetime(s):
                 return datetime.datetime.strptime(
                     s, "%Y-%m-%dT%H:%M:%S.000Z"
                 )  # pragma no cover (This branch was used only when creating a download)
+            elif len(s) == 29 and s.count(' ') == 5:
+                return datetime.datetime.strptime(s, "%a, %d %b %Y %H:%M:%S %Z")
```

Comment:
I am not familiar with this attribute `last_modified`
Could you give an example api?

Also, I will be more convinced if you can add a simple unit tes. You can refer to the contribution [docs](https://github.com/PyGithub/PyGithub/blob/master/CONTRIBUTING.md#automated-tests) for instructions.

Reply:
API here: https://github.com/openworm/openworm_docs/blob/1a3fada7813f42bb9843844b079ad9c48e5e3356/build.py#L175

(which currently uses my fork)

Reply:
FYI there's no test for existing `last_modified` so I won't add a test for the newly added `last_modified_at`.